templates/todo/projects/nodejs-mongo-aks/README.md

```diff
@@ -41,6 +41,14 @@ The following prerequisites are required to use this application. Please ensure
 - [Node.js with npm (18.17.1+)](https://nodejs.org/) - for API backend and Web frontend
 - [Kubernetes CLI (kubectl)](https://kubernetes.io/docs/tasks/tools/)
 
+#### Optional Tools
+
+The following list of tools is optional based on your AKS cluster configuration or your application requirements
+
+- [Helm CLI](https://aka.ms/azure-dev/helm-install) (Required if deploying Helm Charts)
+- [Kustomize CLI](https://aka.ms/azure-dev/kustomize-install) (Required if using Kustomize)
+- [Kubelogin CLI](https://aka.ms/azure-dev/kubelogin-install) (Required for Azure RBAC enabled clusters)
+
```

Comment:
For each of the things that are optional, please leave a troubleshooting note in the read me, so if somebody gets that error, they will know that they need to install that specific tool, for example, if they get in the logs that kubelogin is required. They might search the repo for that error and with that being in the readme, they will know exactly the command to install.

Reply:
We already have this for Helm & kustomize when we detect that your configuration is using Helm or kustomize in your config.  I'll see what we can do for kubelogin since that requires more introspection into the AKS resource.

<img width="863" alt="image" src="https://github.com/Azure/azure-dev/assets/6540159/843f0822-5184-41b1-98bf-dd15e0f38c1a">

Reply:
Missing `kubelogin` when needed should now be better as well.

<img width="865" alt="image" src="https://github.com/Azure/azure-dev/assets/6540159/4163a4d9-add5-45f5-adf7-7827a840c174">